examples/ch6/devops-api/delete.go

```diff
@@ -134,7 +134,7 @@ func deleteDevFrom_DevOps(devops_id string, dev_id string) (bool, error) {
 
 	for i := range developer_operations {
 		if developer_operations[i].Id == devops_id {
-			developer_operations[i].Dev, err = removeDevElement(developer_operations[i].Dev, dev_id)
+			developer_operations[i].Devs, err = removeDevElement(developer_operations[i].Devs, dev_id)
```

Comment:
I assume the switch to plural has some deeper meaning that just a name change?

Reply:
Ryan used Devs in the terraform provider so it's better to change this module struct name as such rather than hunting down every reference of it in the terraform provider.